src/fetch-info.js

```diff
@@ -361,6 +362,7 @@ async function fetchInfoFromIETF(specs, options) {
     return paths.filter(p => p.path.match(/^specs\/rfc\d+\.html$/))
       .map(p => p.path.match(/(rfc\d+)\.html$/)[1]);
   }
+  const httpwgRFCs = await getHttpwgRFCs();
```

Comment:
I'm rolling back on a change I introduced last time to avoid fetching this when that was not needed. I was drunk, the change only resulted in the request being sent for each and every IETF spec that was processed...